bundle-size/test/fixtures/requested_reviewers.json

```diff
@@ -0,0 +1,27 @@
+{
+  "users": [
+    {
+      "login": "jridgewell",
```

Comment:
I assume this the first of multiple reviewers?

Reply:
Correct, a PR with multiple reviewers will have multiple `users` objects